samples/core/iris/iris.py

```diff
@@ -0,0 +1,212 @@
+      model=Channel(type=Model),
+      model_blessing=Channel(type=ModelBlessing)
+  )
+
+  # Uses TFMA to compute an evaluation statistics over features of a model and
+  # perform quality validation of a candidate model (compared to a baseline).
+  # Note: to compile this successfully you'll need TFMA at >= 0.21.5
+  eval_config = tfma.EvalConfig(
+      model_specs=[
+          tfma.ModelSpec(name='candidate', label_key='variety'),
+          tfma.ModelSpec(
+              name='baseline', label_key='variety', is_baseline=True
+          )
+      ],
+      slicing_specs=[tfma.SlicingSpec()],
+      metrics_specs=[
+          tfma.MetricsSpec(
+              metrics=[
+                  tfma.MetricConfig(
+                      class_name='SparseCategoricalAccuracy',
+                      threshold=tfma.config.MetricThreshold(
+                          value_threshold=tfma.GenericValueThreshold(
+                              lower_bound={'value': 0.9}
+                          ),
+                          change_threshold=tfma.GenericChangeThreshold(
+                              direction=tfma.MetricDirection.HIGHER_IS_BETTER,
+                              absolute={'value': -1e-10}
+                          )
+                      )
+                  )
+              ]
+          )
+      ]
+  )
+
+  # Uses TFMA to compute a evaluation statistics over features of a model.
+  model_analyzer = Evaluator(
+      examples=example_gen.outputs['examples'],
+      model=trainer.outputs['model'],
+      baseline_model=model_resolver.outputs['model'],
+      # Change threshold will be ignored if there is no baseline (first run).
+      eval_config=eval_config
+  )
+
+  # Checks whether the model passed the validation steps and pushes the model
+  # to a file destination if check passed.
+  pusher = Pusher(
+      model=trainer.outputs['model'],
+      model_blessing=model_analyzer.outputs['blessing'],
+      push_destination=pusher_pb2.PushDestination(
+          filesystem=pusher_pb2.PushDestination.Filesystem(
+              base_directory=os.path.
+              join(str(pipeline.ROOT_PARAMETER), 'model_serving')
+          )
+      )
+  )
+
+  # TODO(numerology): Add pusher and model validator.
+
+  return pipeline.Pipeline(
+      pipeline_name=pipeline_name,
+      pipeline_root=pipeline_root,
+      components=[
+          example_gen, statistics_gen, infer_schema, validate_stats, transform,
+          trainer, model_resolver, model_analyzer, pusher
+      ],
+      enable_cache=True,
+  )
+
+
+if __name__ == '__main__':
+  absl.logging.set_verbosity(absl.logging.INFO)
+  # Make sure the version of TFX image used is consistent with the version of
+  # TFX SDK. Here we use tfx:0.21.1 image.
+  config = kubeflow_dag_runner.KubeflowDagRunnerConfig(
+      kubeflow_metadata_config=kubeflow_dag_runner.
+      get_default_kubeflow_metadata_config(),
+      tfx_image='tensorflow/tfx:0.21.2',
```

Comment:
OK we retag it later, just the topic of docker.io

Reply:
Ack. Will do that together with other prebuilt sample.